.codacy.yaml

```diff
@@ -0,0 +1,10 @@
+exclude_paths:
+  - '.bundle/**'
+  - 'spec/**/*'
+  - 'benchmarks/**/*'
+  - '*.min.js'
+  - '**/tests/**'
```

Comment:
Wouldn't it be nice to have codacy for tests also?

Reply:
Will remove in next commit.

Reply:
You can mark this as resolved because I can't